.claude/hooks/post-tool-use-tracker.sh

```
@@ -0,0 +1,293 @@
+        detected_skills="skill-generation"
+    elif [[ "$file" =~ /add ]] || [[ "$file" =~ /customize ]] || [[ "$file" =~ /templates/ ]]; then
+        detected_skills="template-library"
+    fi
+
+    echo "$detected_skills"
+}
+
+# Create session file path based on project directory hash
+get_session_file() {
+    local project_dir="$1"
+    local hash=$(echo -n "$project_dir" | md5 2>/dev/null || echo -n "$project_dir" | md5sum | cut -d' ' -f1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)